modules/ossm-observability-access.adoc

```diff
@@ -12,14 +12,16 @@ To access the Kiali console you must have {ProductName} installed and projects c
 
 .Procedure
 
-. Use the perspective switcher to switch to the Administrator perspective. 
+. Use the perspective switcher to switch to the *Administrator* perspective.
 
-. Click *Home* > *Projects*.
+. Click *Home* -> *Projects*.
 
-. Click the name of your project. For example click `bookinfo`.
+. Click the name of your project. For example, click `bookinfo`.
 
-. In the Launcher section, click `Kiali`.
+. In the *Launcher* section, click *Kiali*.
```

Comment:
It looks like some of these were accepted here but marked resolved without incorporating in the above topic; probably best to keep them consistent?

Reply:
I have no idea what this comment means.  I've made these changes, are they not showing up for you?  Maybe a caching issue?